### PR TITLE
Update Free_for_Open_Source_Application_Security_Tools.md

### PR DESCRIPTION
**Pull request by coadaflorin**
Added new entry for HCL AppScan CodeSweep for their new GitHub action.

### CODE REVIEW

File: pages/Free_for_Open_Source_Application_Security_Tools.md [hunk 1]
```diff
@@ -76,7 +76,9 @@ are free for Open Source projects:
   - [Xanitizer](https://www.xanitizer.com/xanitizer-pricing/) - A SAST tool
     for Java, Scala, and JavaScript/TypeScript. Per this pricing page, it
     is free for Open Source projects if you contact the vendor.
-  - [HCL AppScan CodeSweep](https://hclsw.co/codesweep) - This is a SAST community edition version of HCL AppScan. Free for everyone to use. It is delivered as a VS Code plugin and scans files upon saving them. The results show the location of a finding, type and remediation advice. The tool currently supports Python, Ruby, JS (Node, Angular, JQuery, React, etc) , PHP, Perl, Go, TypeScrit & a few more.
+  - [HCL AppScan CodeSweep](https://hclsw.co/codesweep) - This is a SAST community edition version of HCL AppScan. Free for everyone to use. The tool currently supports Python, Ruby, JS (Vue, Node, Angular, JQuery, React, etc), PHP, Perl, Go, TypeScrit & more, with new languages being added frequently.
```
Comment (kingthorin) [inline pages/Free_for_Open_Source_Application_Security_Tools.md:79]:
```suggestion
  - [HCL AppScan CodeSweep](https://hclsw.co/codesweep) - This is a SAST community edition version of HCL AppScan. Free for everyone to use. The tool currently supports Python, Ruby, JS (Vue, Node, Angular, JQuery, React, etc), PHP, Perl, Go, TypeScript & more, with new languages being added frequently.
```

File: pages/Free_for_Open_Source_Application_Security_Tools.md [hunk 2]
```diff
@@ -76,7 +76,9 @@ are free for Open Source projects:
...
+    - [CodeSweep - VS Code Plugin](https://hclsw.co/codesweep) -  Scans files upon saving them. The results show the location of a finding, type and remediation advice.
```
Comment (kingthorin) [inline pages/Free_for_Open_Source_Application_Security_Tools.md:80]:
```suggestion
    - [CodeSweep - VS Code Plugin](https://hclsw.co/codesweep) -  Scans files upon saving them. The results show the location of a finding, type, and remediation advice. 
```

**Comment by coadaflorin** [comment]
@kingthorin thanks for the corrections!

**Review by kingthorin — APPROVED** [review]
Thanks

**Pull request merged**